prow/spyglass/podlogartifact_test.go

```diff
@@ -0,0 +1,301 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spyglass
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"testing"
+
+	"k8s.io/test-infra/prow/spyglass/viewers"
+)
+
+type fakePodLogJAgent struct {
```

Comment:
?

Reply:
added a comment, hopefully that will help :slightly_smiling_face:

Reply:
No, hard to tell what _you_ mean :P